### PR TITLE
Update ruby versions to match currently supported version, remove jruby

### PR DESCRIPTION
**Pull request by Soulou**
JRuby was never used/tested, thus we don't support it officially.

### CODE REVIEW

File: _posts/languages/ruby/2000-01-01-start.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Ruby
 nav: Introduction
-modified_at: 2021-01-05 00:00:00
+modified_at: 2021-41-05 00:00:00
```
Comment (EtienneM) [inline _posts/languages/ruby/2000-01-01-start.md:4]:
```suggestion
modified_at: 2021-05-31 00:00:00
```

File: _posts/languages/ruby/2000-01-01-start.md [hunk 2]
```diff
@@ -77,16 +77,10 @@ $ git push scalingo master
 If your need to install a custom version of Ruby, you can achieve that by
 specifying it in your Gemfile:
 
-### Ruby MRI 2.5.5
+### Ruby MRI 2.7.3
```
Comment (EtienneM) [inline _posts/languages/ruby/2000-01-01-start.md:80]: Did you update our internal documentation explaining how to update the Ruby supported versions to specify to update this doc page?

File: _posts/languages/ruby/2000-01-01-start.md (hunk 1 above)
Comment (Soulou) [reply _posts/languages/ruby/2000-01-01-start.md:4]: Haha I'm really not awake........

File: _posts/languages/ruby/2000-01-01-start.md (hunk 2 above)
Comment (Soulou) [reply _posts/languages/ruby/2000-01-01-start.md:80]: I'm doing it now!

**Comment by Soulou** [comment]
Actually it's already in https://www.notion.so/scalingooriginal/New-Ruby-Release-1c91be8bd30f40d3bbf9d544c3a0b724

**Comment by Soulou** [comment]
However, as support for new ruby version is transparent, sometimes we just miss some

**Comment by Soulou** [comment]
Making a changelog entry months after the release is a bit weird but updating the doc makes sense. What do you think about this situation?

**Review by EtienneM — APPROVED** [review]

**Pull request merged**